CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 * Support for MirrorMaker 1 has been removed
 * Added support to configure `dnsPolicy` and `dnsConfig` using the `template` sections.
 * Store Kafka node certificates in separate Secrets, one Secret per pod.
+* Added support for Strimzi Metrics Reporter to the Kafka brokers/controllers components. 
+  Support for Kafka Connect, MirrorMaker2 and Http Bridge will be added subsequently.
```

Comment:
I would leave this sentence out. It is confusing.
* If you do the others in the scope of 0.46, you will anyway remove this
* If you do them later - e.g. in 0.48 - it will be forever unclear here if it was ever done or not without studying the later release notes.